explainers/navigation-to-bundles.md

```diff
@@ -0,0 +1,83 @@
+# Explainer: Navigation with Web Bundles<br>(a.k.a. Bundled HTTP Exchanges)
+
+Last updated: Oct 29, 2019
+
+(This explainer was originally written and published as: https://docs.google.com/document/d/1KFmtiE3DHgKfQH5-nKtLiacMrXsoKIXQZ-VIMGHMje0/edit?pli=1#)
+
+A [demo video](https://www.youtube.com/watch?v=rs-3R0ji6dA) that tries to show one of the use-case scenarios of Web Bundles (a.k.a. Bundled HTTP Exchanges):
+
+<a href="https://www.youtube.com/watch?v=rs-3R0ji6dA">
+<img src="https://img.youtube.com/vi/rs-3R0ji6dA/0.jpg">
+</a>
+
+Today, loading a website basically means retrieving multiple resources from one or multiple web servers. This gives web a great strength of linkable, indexable, composable and ephemeral, but it is also making it difficult for a website to:
+
+* be shared and loaded in a self-contained way, similar to what packaged apps and other portable data representation formats provide,
+* be composed as a distributable, installable application, or
+* load reliably and quickly even when a site consists of a large number of subresources, and even when the internet access is limited.
+
+Instead, imagine if we could bundle up a “full” website in a single resource file, so that the website could be shared via a SD card or over some p2p protocol, or could be retrieved from a fast cache or a nearby proxy. It would open up various interesting use cases.
```

Comment:
I think you can drop the quotes around "full". This phrasing doesn't imply that you *have to* include the full website, just that it's possible.

```suggestion
Instead, imagine if we could bundle up a full website in a single resource file, so that the website could be shared via a SD card or over some p2p protocol, or could be retrieved from a fast cache or a nearby proxy. It would open up several interesting use cases.
```